python/ray/_private/services.py

```diff
@@ -1766,6 +1788,50 @@ def build_cpp_worker_command(
 
     return command
 
+def build_julia_worker_command(
+    bootstrap_address: str,
+    plasma_store_name: str,
+    raylet_name: str,
+    redis_password: str,
+    session_dir: str,
+    log_dir: str,
+    node_ip_address: str,
+    setup_worker_path: str,
+):
+    """This method assembles the command used to start a Julia worker.
+
+    Args:
+        bootstrap_address: Bootstrap address of ray cluster.
+        plasma_store_name: The name of the plasma store socket to connect
+           to.
+        raylet_name: The name of the raylet socket to create.
+        redis_password: The password of connect to redis.
+        session_dir: The path of this session.
+        node_ip_address: The ip address for this node.
+        setup_worker_path: The path of the Python file that will set up
+            the environment for the worker process.
+    Returns:
+        The command string for starting Julia worker.
+    """
+
+    # TODO: --project, --startup-file=no, --banner=no, ...?
+    command = [
+        sys.executable,
+        setup_worker_path,
+        DEFAULT_WORKER_EXECUTABLE,
+        f"--ray_plasma_store_socket_name={plasma_store_name}",
+        f"--ray_raylet_socket_name={raylet_name}",
+        "--ray_node_manager_port=RAY_NODE_MANAGER_PORT_PLACEHOLDER",
+        f"--ray_address={bootstrap_address}",
+        f"--ray_redis_password={redis_password}",
+        f"--ray_session_dir={session_dir}",
+        f"--ray_logs_dir={log_dir}",
+        f"--ray_node_ip_address={node_ip_address}",
+        "RAY_WORKER_DYNAMIC_OPTION_PLACEHOLDER",
```

Comment:
we'll want to setup our entry point for the worker runtime to handle these args right?

Reply:
correct - as in https://github.com/beacon-biosignals/ray_core_worker_julia_jll.jl/pull/17/files